app/models/item.rb

```diff
@@ -4,6 +4,7 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
   has_many :order_items
   has_attached_file :image, styles: { small: "64x64", med: "100x100", large: "200x200"}, default_url: "http://static.neatorama.com/images/2014-04/bazooka-vespa.jpg"
+  validates_attachment_content_type :image, content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"]
```

Comment:
This is slick. Didn't know you could pass it an array of acceptable attachment types.